config/src/config/storage_config.rs

```diff
@@ -494,16 +500,27 @@ impl ConfigSanitizer for StorageConfig {
             .ledger_pruner_config
             .user_pruning_window_offset;
 
-        if ledger_prune_window < 50_000_000 {
-            warn!("Ledger prune_window is too small, harming network data availability.");
+        // Sanitize the prune window configs
```

Comment:
@msmouse @grao1991 are these fine?

Reply:
No..? when we run benchmarks / forge etc we do want to test aggressive pruner settings to review bugs. Or for emergency we want to release disk space aggresively. I'd keep those warns.

Reply:
+1

Reply:
Gotcha -- I wasn't sure how you folks felt (hence why I made this a separate commit). I'll just delete the commit from the stack. Note: I'm not sure how useful warnings are in these cases (I really don't think they'll stop anyone from using bad configs like this), but happy to defer to you 😄